airbyte-db/db-lib/src/main/resources/sql_functions_and_triggers/config_events.sql

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
This is used as part of a migration and I think it should be highlighted. I think it is worth documenting and would probably rename this file to contain the migration version number.